provider/pkg/schema/gen.go

```diff
@@ -1482,9 +1482,8 @@ func (ctx *cfSchemaContext) genEnumType(enumName string, propSchema *jsschema.Sc
 			continue
 		}
 
-		// Special case for `ChannelPreset` enum in the `AWS::IVS::Channel` resource. The enum has an empty string value which
-		// gets returned by the service in certain default cases (for channel types (BASIC and STANDARD)).
-		if typName == "ChannelPreset" && str == "" {
+		// Special case for when there is an enum which has an empty string value
```

Comment:
Is there a linked TODO?

Reply:
No, but good call. We should automate this case.